tailwind.config.js

```diff
@@ -1,11 +1,37 @@
+const colors = require('tailwindcss/');
+
 module.exports = {
   purge: ['./src/**/*.{js,jsx,ts,tsx}', './public/index.html'],
   darkMode: false, // or 'media' or 'class'
   theme: {
-    extend: {},
+    extend: {
+      main: '#050005',
+      frame: '#1c3d46',
+      grey: '#474c52',
+      purple: '#4D194D',
+      white: '#E9ECEF',
```

Comment:
All these should be constants. do something like this:

```
enum COLORS {
  MAIN: '#050005',
  ...
}
```

and replace all instances in this page with their correct color.

Btw, this is how you create enums in Typescript: https://www.typescriptlang.org/docs/handbook/enums.html. They are constants but better